filer/admin/clipboardadmin.py

```diff
@@ -69,6 +76,22 @@ def get_model_perms(self, *args, **kwargs):
         }
 
 
+@csrf_exempt
+def check_file_constraints(request, folder_id=None):
```

Comment:
Any reason this is a function view not a class-based view? It's just that I imagine these will eventually get deprecated, so I would have thought it makes sense to keep to the newer way of writing views, right?